src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+#server port
+server.port=8207
+#mongodb
+spring.data.mongodb.uri=mongodb://${DB_USER:admin}:${DB_PASSWORD:pwd}@${DB_HOST:localhost}:${DB_PORT:27017}/${DB_NAME:forms}
+#logging
+logging.level.org.springframework.data=debug
+logging.level.root=INFO
+logging.level.uk.nhs.hee.trainee.details=TRACE
```

Comment:
Should it be 'trainee.forms' instead of 'trainee.details'?

Reply:
Yeah, will change it.